things 14, 16 and 18 available for review

rowan-ardc opened the pull request.

rowan-ardc committed:
Create thing-14.md

rowan-ardc committed:
Replace html with Matthias's processed markdown

rowan-ardc committed:
Create thing-16.md

rowan-ardc committed:
Create thing-18.md

rowan-ardc committed:
Remove html tags

rowan-ardc committed:
Format heading

rowan-ardc committed:
Add badge image file for thing 14

rowan-ardc committed:
Add credly badge links

rowan-ardc committed:
Add link to thing 14

rowan-ardc committed:
Remove html tags

rowan-ardc committed:
Add funders slide

rowan-ardc committed:
Add badge for thing 16

rowan-ardc committed:
Fix image links

rowan-ardc committed:
Add link to thing 16

rowan-ardc committed:
Check hyperlinks

rowan-ardc committed:
Remove html tags

rowan-ardc committed:
Add badge for thing 18

rowan-ardc committed:
Add link to thing 18

rowan-ardc committed:
Fix headings

rowan-ardc committed:
Add top level menu

rowan-ardc committed:
Delete note

At head of 'challenge me' 
"Note: if you aren’t familiar with Data Curation Profiles, you may wish to skim over the Learn more Option 2 activity before you start this one."

rowan-ardc committed:
Update slack & twitter

rowan-ardc committed:
Edit hashtag

rowan-ardc committed:
Update slack and twitter

rowan-ardc committed:
Update slack and twitter

ragamouf committed:
challenge me thing 14

just wanted to address the slight inconsistency between iv and tutorial 4, and ended up rewriting the **If you have time** section.

ragamouf committed:
ordered list finagles

ragamouf committed:
Update thing-14.md

ragamouf commented:
er - I made edits to your thing 14 over in your branch...! Sorry! May need to re do the commit?

ragamouf commented:
Hi @rowan-ardc, the link on Thing 16 for Dryad goes to a wiki page on the Joint Data ARchiving Policy. Is this still current and useful to link to? I think Dryad's business model has changed significantly since 23 things, so there might be another more appropriate page to link to?

rowan-ardc committed:
Replace policy link

Replaced Dryad with Elsevier, under the "Choice 1: Journal data policies" section.  Relates to @ragamouf [comment](https://github.com/au-research/ARDC-23-things/pull/13#issuecomment-566822008)

rowan-ardc commented:
Hi @ragamouf - I replaced the dryad link with a pointer to the Elsevier policy.  Any further thoughts on this or suggested alternatives?
> 
> 
> Hi @rowan-ardc, the link on Thing 16 for Dryad goes to a wiki page on the Joint Data ARchiving Policy. Is this still current and useful to link to? I think Dryad's business model has changed significantly since 23 things, so there might be another more appropriate page to link to?

rowan-ardc commented:
Thanks @ragamouf  - looks fine
> 
> 
> er - I made edits to your thing 14 over in your branch...! Sorry! May need to re do the commit?

mpfl reviewed (changes requested):
Could you please add links to the previous thing? Sorry, my example in Thing 1 didn't have this because there is no Thing 0!

rowan-ardc committed:
Fix navigation

rowan-ardc committed:
Fix navigation again

rowan-ardc committed:
Fix navigation

rowan-ardc committed:
Fix navigation

rowan-ardc commented:
righto - have fixed navigation
> 
> 
> Could you please add links to the previous thing? Sorry, my example in Thing 1 didn't have this because there is no Thing 0!

rowan-ardc committed:
Remove credly

rowan-ardc committed:
Remove credly

rowan-ardc committed:
Remove credly

rowan-ardc committed:
Delete Badge-thing14.png

rowan-ardc committed:
Delete Badge-thing16.png

rowan-ardc committed:
Delete Badge-thing18.png

rowan-ardc commented:
Removed credly text & badge files

NicholaBurton commented on things/thing-14.md:
```diff
@@ -0,0 +1,179 @@
+- Tweet to [@ardc_au](https://twitter.com/ARDC_AU) using hashtag #23things
+
+Go to [Thing 15](thing-15.md "thing 15")
+Data Management Plans [All Things](index.md)
```
need an "or to" before All Things (here and throughout)

rowan-ardc replied:
Fixed each occurrence

NicholaBurton commented on things/thing-16.md:
```diff
@@ -0,0 +1,167 @@
+    Trust](https://wellcome.ac.uk/funding/managing-grant/policy-data-management-and-sharing)
+    in the UK (1 page)
+
+Now, imagine it is 2020… **consider** what you think Australian research
```
Pretty soon this won't take much imagination! We might need to update the date... 2025?

rowan-ardc replied:
Changed date to 2025

NicholaBurton commented on things/thing-18.md:
```diff
@@ -0,0 +1,177 @@
+- Tweet to [@ardc_au](https://twitter.com/ARDC_AU) using hashtag #23things
+
+Go to [Thing 19](thing-19.md "thing 19")
+Exploring APIs andApps or to [All Things](index.md)
```
missing space in "and Apps" here and throughout

rowan-ardc replied:
Fixed each occurrence

NicholaBurton reviewed (changes requested):
I notice that these Things are missing the Thing Theme overview that is at the bottom of some of the others (i.e. Things 16 and 18 are part of the theme "Let's talk data"). Should we add this section to all things for consistency?

rowan-ardc committed:
Add "metadata & more" group minus badge

rowan-ardc committed:
Add "Let's talk data" group minus badge

rowan-ardc committed:
Add "Lets talk data" group minus badge

rowan-ardc committed:
Fix navigation syntax

In each occurrence of the following, inserted "or to"
Go to [Thing 15](thing-15.md "thing 15") Data Management Plans or to [All Things](index.md)

rowan-ardc commented:
Reflected in Matthias's example.  I've reinstated to 14, 16 and 18.

> 
> 
> I notice that these Things are missing the Thing Theme overview that is at the bottom of some of the others (i.e. Things 16 and 18 are part of the theme "Let's talk data"). Should we add this section to all things for consistency?

rowan-ardc committed:
Speculative date becomes 2025

rowan-ardc committed:
Minor edit to separate two words

rowan-ardc commented:
requested changes complete

tom-h committed:
Merge branch 'gh-pages' into gh-pages
